qgispluginci/__about__.py

```diff
@@ -47,13 +54,34 @@
 __uri_tracker__ = f"{__uri__}issues/"
 
 # This string might be updated on CI on runtime with a proper semantic version name with X.Y.Z
-__version__ = "__VERSION__"
+__version__ = getenv("GITHUB_REF_NAME", "__VERSION__")
 
 if "." not in __version__:
     # If __version__ is still not a proper semantic versioning with X.Y.Z
     # let's hardcode 0.0.0
     __version__ = "0.0.0"
 
+    # try to get it from git tag
+    try:
+        git_cmd = "git tag -l --sort=-creatordate | head -n 1"
+        returned_output: bytes = subprocess.check_output(git_cmd, shell=True)
+        if len(returned_output):
+            __version__ = returned_output.decode("utf-8")
+        else:
+            chglog = ChangelogParser(parent_folder=Path(__file__).parent.parent)
```

Comment:
Inception, using our own code to build the metadata ;-)

Reply:
With this, Python is definitely a :snake:, eating its own code!